sql/resource_sql_alerts.go

```diff
@@ -12,7 +12,7 @@ import (
 type AlertOptions struct {
 	Column        string `json:"column"`
 	Op            string `json:"op"`
-	Value         string `json:"value"`
+	Value         any    `json:"value"`
```

Comment:
I suspect we probably can't directly use `any` here without other changes down the line.

Reply:
We can't change type of the existing resource. See this PR on how would you handle it: https://github.com/databricks/terraform-provider-databricks/commit/b2208bb8c941d2519359e3ac3dc196206c5b7c99

Reply:
This didn't work indeed. I think I have a solution for now. It's a pity that we don't have union types in OpenAPI (e.g. string or number).